### ISSUE TITLE
Abl-imatinib example in explicit solvent

### ISSUE THREAD

**Issue by andrrizzi** [choderalab/yank#252]
I guess this also is related to #168 .

In this example, with the current default alchemical protocol for explicit solvent and complex the Markov chain is decomposable. The state mixing transition matrix after 1000 iterations for the complex is
```
0     1     2     3     4     5     6     7     8     9    10    11    12
1.000
      0.698 0.250 0.053
      0.250 0.532 0.212
      0.053 0.212 0.683 0.053
                  0.053 0.888 0.052
                        0.052 0.900
                                    0.912
                                          0.937
                                                0.950
                                                      0.915 0.052
                                                      0.052 0.829 0.109
                                                            0.109 0.809 0.074
                                                                  0.074 0.916
```
with protocol
```python
def defaultComplexProtocolExplicit(cls):
    ...
    alchemical_states.append(AlchemicalState(1.00, 1.00, 1.00, 1.)) # fully interacting
    alchemical_states.append(AlchemicalState(1.00, 0.00, 1.00, 1.)) # discharged
    alchemical_states.append(AlchemicalState(1.00, 0.00, 0.95, 1.))
    alchemical_states.append(AlchemicalState(1.00, 0.00, 0.90, 1.))
    ...
    alchemical_states.append(AlchemicalState(1.00, 0.00, 0.00, 1.)) # discharged, LJ annihilated
```
the solvent is in the same situation.

**Comment by jchodera** [comment]
Fixed by #254, though alchemical protocol likely needs more tuning.

**Comment by jchodera** [comment]
The #254 fix also fixes the MPI capabilities, so you can now use parallel MPI runs using the included `run-torque.sh` script.
